specs-go/config.go

```diff
@@ -178,7 +178,8 @@ type Linux struct {
 	// MountLabel specifies the selinux context for the mounts in the container.
 	MountLabel string `json:"mountLabel,omitempty"`
 	// IntelRdt contains Intel Resource Director Technology (RDT) information for
-	// handling resource constraints (e.g., L3 cache, memory bandwidth) for the container
+	// handling resource constraints (e.g., L3 cache, memory bandwidth) and monitoring (eg. MBM, CMT)
```

Comment:
How about:
"// handling resource constraints and monitoring (e.g., L3 cache, memory bandwidth) for the container"?